decidim-core/lib/decidim/core.rb

```diff
@@ -845,6 +845,12 @@ def self.organization_settings(model)
     0.seconds
   end
 
+  # Public Setting that allows to configure which component will have "Use my current location" button
+  # in a geocoded address field. Accepts an array of component manifest names or simply the value ":all" to apply everywhere (default value)
+  config_accessor :show_my_location_button do
+    :all
```

Comment:
Setting this as plain symbol raises an error..,

```suggestion
    [:all]
```